### DISCUSSION TITLE
First User Feedback

### DISCUSSION THREAD

**Discussion by AlexanderThiele** [btwld/mix#522] [General]
Hey,

I've just used the package for the first time and I found it quite difficult to get started. 

I think it's a great idea that you can create the styling independently of the widgets. 

My first attempt was to define the styles and simply use the defined attributes in my widgets. I reached the limits of the package and got lost in the dokus. 

So my first question is: why is it so difficult to use the style attributes in normal widgets?

**Comment by tilucasoli** [comment]
Hey @AlexanderThiele, thank you for your feedback.
When you refer to "normal" widgets, I assume you're talking about components that aren’t part of Mix. Answering your question, it happens because we have a lot of logic behind mix to make it functional so all this magic is embed in our widgets.

I would love to help you with all your questions about mix and improve your experiences with it. Could you share them with me and give us the examples you are trying to do?
